src/NServiceBus.Transport.Msmq.Tests/InstanceMapping/InstanceMappingFileFeatureTests.cs

```diff
@@ -11,13 +11,15 @@ public class InstanceMappingFileFeatureTests
         [Test]
         public void Should_configure_default_values()
         {
-            var feature = new InstanceMappingFileFeature();
-            var settings = new SettingsHolder();
+            // Test has to be commented out for now, no way to access the defaults
```

Comment:
@Particular/nservicebus-maintainers This is the one place where I couldn't figure out a workaround, so I had to just comment out the test for now.

`ConfigureDefaults` is internal, and derived types have no access to `registeredDefaults`, so I couldn't pull off a similar change like I did for `PerformStartup`.

I'm not sure we want to expose this to features, so maybe this test just needs to be deleted?